### ISSUE TITLE
Shadowmourne quest, LK chest items and the quest "Trial of the Naaru: Tenacity"

### ISSUE THREAD

**Issue by sowhat2** [dalaranwow/dalaran-wow#5872]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
       For the Shadowmourne quest after killing LK, i got the chest from LK, i can't turn in the othe items to complete the future quests for the items.
For the quest in Arcatraz, "Trial of the Naaru: Tenacity" tried to compelte it 3 times, in 3 days, and after killing the boss, the quest won't compelete. In this quest Millhouse Manastorm should help you in the fight, but instead he is doing nothing.
**Expected behaviour**: 
         I should be able to talk with every alliance/horde hero in order to get the quest and compelete, for the items.
Millhouse Manastorm should join the fight.

**Steps to reproduce the problem**: In the case of the shadowmourne quest, i think its a known problem.
           In the case of "Trial of the Naaru:Tenacity" don't know what else i can say, maybe just that if the boss does mind control on you, after 5 sec you won't be able to move at all, you are stuck in same position. Don't know if this is the case of a one man party.

1. 
2. 
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Golganeth** [comment]
Greetings!
Please when you open a bug report, do it separately (one report = one bug).
- The problem you mention with the Shadowmourne quest, it's a well known and general issue with quests that require materials. Abandon and retake the quest, and you will be able to turn it in just fine.
- The problem with the quest "Trial of the Naaru: Tenacity" is already reported. See #2666 / #5859 for reference.

Closing.

**Comment by sowhat2** [comment]
I don't have what to abbandon with the shadowmourne quest chain. After i compelete the quest The Lich King's Last Stand, i get the item from Lich King, "Sealed Chest". After i turn this in to Highlord Mograine, i get the mount and few more items. Then normaly i would nee to go talk with jaina, silvanas, muradin, king teranas and they will give me a quest to turn those items in and the reward from them lightbringer tabar and so on...i can't turn in those items because there is no quest ! on their NPC.
